README.md

```diff
@@ -31,4 +31,4 @@ Run `yarn install`. You'll also need `jekyll` to run a local host. Run `gem inst
   - for each graph, add a div with a unique id
 - add a `render<topic name>` function in [`main.ts`](main.ts)
   - the logic to generate the graphs should be included in this function
-- link the section in [`introduction.html`](./content/introduction.html), Table of contents
+- link the section in [`introduction.html`](./content/introduction.html), Sections (previously Table of Contents)
```

Comment:
no need to mention what it was previously imo? i think it's understandable if we just write `Sections` (documentation for ourselves)